src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyPrice = 40;
+  const totalCost = dailyPrice * days;
+  const longTerm = 7;
+  const shortTerm = 3;
+  const longTermDiscount = 50;
+  const shortTermDiscount = 20;
+
+  if (days >= longTerm) {
+    return days * dailyPrice - longTermDiscount;
```

Comment:
If the number of days is greater than or equal to the long-term threshold, the long-term discount is applied. However, this logic does not account for cases where both discounts could be applicable. Consider revising the logic to ensure that the correct discount is applied based on the number of days.